source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalifications.cs

```diff
@@ -0,0 +1,72 @@
+﻿using CommonJobs.Application.EvalForm.DTOs;
+using CommonJobs.Domain.Evaluations;
+using CommonJobs.Infrastructure.RavenDb;
+using Raven.Client.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace CommonJobs.Application.EvalForm.Commands
+{
+    public class GetEvaluationCalifications : Command<CalificationsDTO>
+    {
+        private string _period { get; set; }
+
+        private string _evaluatedUser { get; set; }
+
+        private string _loggedUser { get; set; }
+
+        public GetEvaluationCalifications(string period, string evaluatedUser, string loggedUser)
+        {
+            _period = period;
+
+            _loggedUser = loggedUser;
```

Comment:
The idea was to avoid having logic in the controllers. I understand your point and will try to rethink it a little.